servicecontrol/configure-ravendb-location.md

```diff
@@ -4,11 +4,9 @@ summary: Increase space for monitored data by configuring ServiceControl to save
 reviewed: 2016-11-09
 ---
 
-Each ServiceControl service stores its data in a RavenDB embedded database. The location of the database is set at install time.
+Each standard ServiceControl instance stores its data in a RavenDB embedded database. The location of the database is set at install time.
```

Comment:
We need to differentiate between Monitoring and non-Monitoring instances. In the SCMU we refer to them as ServiceControl instances and Monitoring instances. Maybe we need to further differentiate the existing ServiceControl instances with a different name?

Maybe
- Ingestion instance
- Recoverability instance
- Data instance
???